app/controllers/admins_controller.rb

```diff
@@ -7,5 +7,14 @@ class AdminsController < ApplicationController
 
   def show
     authorize! :admin_dashboard
+    @presenter = build_presenter
+
+    @presenter.work_stats = StatBuilder.build_stats if user_with_groups.administrator?
+  end
+
+  private
+
+  def build_presenter
```

Comment:
I'm wondering if having this as a separate method is useful.  We're not encapsulating a lot of complexity or reusing this, so I'd say it's easier to read the code without it.

Reply:
Do we anticipate needing to build out the presenter for the admin dashboard as more functionality is added there? That was the reason I kept using the `build_presenter` method. But I'm fine to remove if we're not needing that for now.

Reply:
I'd remove it until we need it. It's really not much work to create a method.